Makefile

```diff
@@ -104,8 +107,8 @@ collect-raw:
 	--benchmark faux
 
 collect-data:
-	@python python/kernmlops collect -v \
-	-c ${KERNMLOPS_CONFIG_FILE}
+	python python/kernmlops collect -v \
+	-c ${KERNMLOPS_CONFIG_FILE} -p "$(COLLECTION_PREFIX)"
```

Comment:
If someone doesn't specify a collection prefix won't this crash?

Reply:
it will not. It will expand to `""` and `-p ""` is equivalent to not passing the argument at all. I have tested it